### PR TITLE
Add new dependency: com.squareup.okhttp3:okhttp

### PR DESCRIPTION
**Pull request by snjeza**
Add new dependency: com.squareup.okhttp3:okhttp - https://github.com/eclipse/che/pull/4363

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -45,6 +45,7 @@
         <com.googlecode.gson.version>2.3.1</com.googlecode.gson.version>
         <com.h2database.version>1.4.192</com.h2database.version>
         <com.jcraft.jsch.version>0.1.53</com.jcraft.jsch.version>
+        <com.squareup.okhttp3>3.4.2</com.squareup.okhttp3>
```
Comment (amisevsk) [inline pom.xml]: Could we use `com.square.okhttp3.version` to match the other variables?

**Review by amisevsk — COMMENTED** [review]
LGTM. 3.4.2 is the version used in kubernetes-client 2.1.1.

**Review by skabashnyuk — APPROVED** [review]

**Comment by skabashnyuk** [comment]
@amisevsk can you make sure that 
if this transitive dependency is not used or approved over CQ
```<dependency>
<groupId>com.squareup.okio</groupId>
<artifactId>okio</artifactId>
</dependency>
```

**Comment by amisevsk** [comment]
@skabashnyuk We are currently updating dependencies in the openshift-connector branch to avoid having to go through the CQ process too frequently. As Che issue #[2847](https://github.com/eclipse/che/issues/2847) progresses, we'll probably have to update these same dependencies again (e.g. if we need to move to a newer version of kubernetes-client). 

Once we open a PR to merge the openshift-connector branch into master, we'll go through and ensure CQs are opened on all updated transitive dependencies.

**Comment by skabashnyuk** [comment]
ok

**Pull request merged**